zookeeper-server/src/main/java/org/apache/zookeeper/server/DataTree.java

```diff
@@ -300,6 +303,20 @@ public DataTree() {
 
         addConfigNode();
 
+        /**
+         * throw warning if child node count is more the maxChildCountThreshold
+         * this will help to detect overpopulated znodes
+         */
+        String mcct = System.getProperty("zookeeper.maxChildCountThreshold");
+        if (mcct != null && !mcct.isEmpty()) {
```

Comment:
- add a doc to `zookeeperAdmin.md` for this new property
- Make the parse of property concise. e.g: 

> private static final long rateLimiterInterval = Integer.parseInt(System.getProperty(ADMIN_RATE_LIMITER_INTERVAL, "300000"));